foundations/foundation-config/src/main/java/org/apache/servicecomb/config/ConfigUtil.java

```diff
@@ -181,6 +183,14 @@ private static void duplicateServiceCombConfigToCse(ConcurrentCompositeConfigura
     compositeConfiguration.addConfiguration(source, sourceName);
   }
 
+  private static void duplicateServiceCombConfigToCseAtFront(ConcurrentCompositeConfiguration compositeConfiguration,
+      AbstractConfiguration source,
+      String sourceName) {
+    duplicateServiceCombConfigToCse(source);
```

Comment:
duplicateServiceCombConfigToCse has already addConfiguration,  and here addConfigurationAtFront again, is it necessary to add it twice?

Reply:
mapping
ENV: servicecomb.abc
we can use cse.abc to get the value